proposed/openebs/v0.7/README.md

```diff
@@ -0,0 +1,116 @@
+------------------------------------------------------------------------------
```

Comment:
Hi @mukeshmk 
Can you remove the Deprecation notice since this only applies to the chart hosted in OpenEBS repo.